pkgs/applications/version-management/git-review/default.nix

```
@@ -1,26 +1,38 @@
-{ lib, fetchurl, buildPythonApplication, pbr, requests, setuptools }:
+{ lib
+, fetchurl
+, buildPythonApplication
+, pbr
+, requests
+, setuptools
+}:
 
 buildPythonApplication rec {
   pname = "git-review";
-  version = "2.0.0";
+  version = "2.1.0";
 
   # Manually set version because prb wants to get it from the git
   # upstream repository (and we are installing from tarball instead)
   PBR_VERSION = version;
 
   src = fetchurl {
     url = "https://opendev.org/opendev/${pname}/archive/${version}.tar.gz";
-    sha256 = "0dkyd5g2xmvsa114is3cd9qmki3hi6c06wjnra0f4xq3aqm0ajnj";
+    hash = "sha256-3A1T+/iXhNeMS2Aww5jISoiNExdv9N9/kwyATSuwVTE=";
   };
 
-  propagatedBuildInputs = [ pbr requests setuptools ];
+  propagatedBuildInputs = [
+    pbr
+    requests
+    setuptools # implicit dependency, used to get package version through pkg_resources
+  ];
 
-  # Don't do tests because they require gerrit which is not packaged
+  # Don't run tests because they pull in external dependencies
+  # (a specific build of gerrit + maven plugins), and I haven't figured
+  # out how to work around this yet.
   doCheck = false;
```

`pythonImportsCheck` is an additional safe-guard.

```suggestion
  # out how to work around this yet.
  doCheck = false;
  
  pythonImportsCheck = [ "git_review" ];
```

---

Thanks, I'll add this.